src/main/steps/common/respondent-summary/template.njk

```diff
@@ -35,7 +35,9 @@
             {{forRecords}}
           </p>
 
-          <a class="govuk-button ga-pageLink govuk-button--secondary" role="button" draggable="false" data-module="govuk-button" data-ga-category="check your answers" data-ga-label="download draft" download="" href="/tasklistresponse/generate-c7-draft">{{downloadDraft}}</a>
+          <a class="govuk-button ga-pageLink govuk-button--secondary" role="button" draggable="false" data-module="govuk-button" data-ga-category="check your answers" data-ga-label="download draft" download="" href="/tasklistresponse/generate-c7-draft/en">{{downloadDraft}}</a>
+          <br/>
+          <a class="govuk-button ga-pageLink govuk-button--secondary" role="button" draggable="false" data-module="govuk-button" data-ga-category="check your answers" data-ga-label="download draft welsh" download="" href="/tasklistresponse/generate-c7-draft/cy">Download draft response welsh</a>
```

Comment:
We need to show this button only when case level welsh language requirement is present

Reply:
Added condition to show and hide button on welsh language requirement